includes/class-wcpdf-admin.php

```diff
@@ -426,101 +428,110 @@ public function output_number_date_edit_fields( $document, $data ) {
+									<p>
+										<span><strong><?= $data['date']['label']; ?></strong></span>
+										<span><?= $data['date']['formatted']; ?></span>
+									</p>
+								</p>
+							</div>
+							<?php endif; ?>
+							<?php do_action( 'wpo_wcpdf_meta_box_after_document_data', $document, $document->order ); ?>
+						<?php else : ?>
+							<span class="wpo-wcpdf-set-date-number button"><?php printf( __( 'Set %s number & date', 'woocommerce-pdf-invoices-packing-slips' ), $document->get_title() ); ?></span>
 						<?php endif; ?>
-						<?php do_action( 'wpo_wcpdf_meta_box_after_document_data', $document, $document->order ); ?>
-					<?php else : ?>
-						<span class="wpo-wcpdf-set-date-number button"><?php printf( __( 'Set %s number & date', 'woocommerce-pdf-invoices-packing-slips' ), $document->get_title() ); ?></span>
-					<?php endif; ?>
-				</div>
+					</div>
 
-				<!-- Editable -->
-				<div class="editable">
-					<?php if( isset( $data['number'] ) ) : ?>
-					<p class="form-field <?= $data['number']['name']; ?>_field ">
-						<label for="<?= $data['number']['name']; ?>"><?= $data['number']['label']; ?></label>
-						<input type="text" class="short" style="" name="<?= $data['number']['name']; ?>" id="<?= $data['number']['name']; ?>" value="<?= $data['number']['plain']; ?>" disabled="disabled" > (<?= __( 'unformatted!', 'woocommerce-pdf-invoices-packing-slips' ) ?>)
-					</p>
-					<?php endif; ?>
-					<?php if( isset( $data['date'] ) ) : ?>
-					<p class="form-field form-field-wide">
-						<label for="<?= $data['date']['name'] ?>[date]"><?= $data['date']['label']; ?></label>
-						<input type="text" class="date-picker-field" name="<?= $data['date']['name'] ?>[date]" id="<?= $data['date']['name'] ?>[date]" maxlength="10" value="<?= $data['date']['date']; ?>" pattern="[0-9]{4}-(0[1-9]|1[012])-(0[1-9]|1[0-9]|2[0-9]|3[01])" disabled="disabled"/>@<input type="number" class="hour" disabled="disabled" placeholder="<?php _e( 'h', 'woocommerce' ); ?>" name="<?= $data['date']['name']; ?>[hour]" id="<?= $data['date']['name']; ?>[hour]" min="0" max="23" size="2" value="<?= $data['date']['hour']; ?>" pattern="([01]?[0-9]{1}|2[0-3]{1})" />:<input type="number" class="minute" placeholder="<?php _e( 'm', 'woocommerce' ); ?>" name="<?= $data['date']['name']; ?>[minute]" id="<?= $data['date']['name']; ?>[minute]" min="0" max="59" size="2" value="<?= $data['date']['minute']; ?>" pattern="[0-5]{1}[0-9]{1}"  disabled="disabled" />
-					</p>
-					<?php endif; ?>
-				</div>
-			</section>
-
-			<!-- Document Notes -->
-			<?php if( array_key_exists( 'notes', $data ) ) : ?>
-
-				<?php do_action( 'wpo_wcpdf_meta_box_before_document_notes', $document, $document->order ); ?>
-
-				<section class="wcpdf-data-fields-section notes">
-					<p class="form-field form-field-wide">
-						<div>
-							<span><strong><?= $data['notes']['label']; ?></strong></span>
-							<span class="wpo-wcpdf-edit-document-notes dashicons dashicons-edit"></span>
```

Comment:
Did you remove this by accident? I can no longer edit the notes without editing the number and date, which means that the invoice number format may change for example.

Reply:
I thought that in terms of experience would be better for the user to just click one time and have all the options there, but I can revert that.

Reply:
clicking edit for the document should reveal both notes and numbers, clicking edit for the notes should reveal only the notes. I thought it was like that in the current release, if not, it should be.

Reply:
Ok, I just checked, and indeed, the notes requires an additional click in the current release. That wasn't the point of my comment though, rather that you should be able to add a note without resetting number formatting.

Reply:
> clicking edit for the document should reveal both notes and numbers, clicking edit for the notes should reveal only the notes. I thought it was like that in the current release, if not, it should be.

That's how it is working in the last changes.